samples/ModbusMaster/component.mk

```diff
@@ -0,0 +1,2 @@
+ARDUINO_LIBRARIES = ModbusMaster
+COMPONENT_INCDIRS = include ../../Sming/Libraries/ModbusMaster/src/
```

Comment:
That line should be changed. 
1. The ModbusMaster library should define its include directories.

Sming/Libraries/ModbusMaster/component.mk
```
COMPONENT_INCDIRS := src/ 
```




2. In the sample you can leave `include` only.

samples/ModbusMaster/component.mk
```
ARDUINO_LIBRARIES = ModbusMaster
COMPONENT_INCDIRS := include
```